routes/addBook.js

```diff
@@ -1,9 +1,48 @@
-const express = require('express')
-const router = express.Router()
+const express = require("express");
+const router = express.Router();
+const Book = require("../models/bookModel");
 
-router.post('/addbook', (req, res) => {
-    console.log(req.body);
-    
-})
+function isValid(book) {
+  return (
+    book.bookTitle &&
+    book.bookTitle.toString().trim() !== "" &&
+    book.wordGoals &&
+    book.wordGoals.toString().trim() !== ""
+  );
+}
 
-module.exports = router;
\ No newline at end of file
+router.post("/addbook", (req, res) => {
+  if (isValid(req.body)) {
+    //Store in the DB
+    const book = new Book({
+      booktitle: req.body.bookTitle,
+      wordgoals: req.body.wordGoals,
+      deadline: req.body.deadline
+    });
+
+    book
```

Comment:
You can validate the book first by doing `book.validate()`
See here: https://mongoosejs.com/docs/documents.html#validating